scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
Hm, good point. Let me see if I get this right:

`project:releng:balrog:server:release` => `release`,
`project:releng:balrog:server:esr` => `esr`, (I could add a separate dict containing esr45 + esr52)
`project:releng:balrog:server:beta` => `beta`,
`project:releng:balrog:server:release` => `nightly` => `all-nightly-branches`

?